apps/emqx_bridge_mongodb/test/emqx_bridge_mongodb_SUITE.erl

```diff
@@ -157,9 +168,17 @@ start_apps() ->
     %% we want to make sure they are loaded before
     %% ekka start in emqx_common_test_helpers:start_apps/1
     emqx_common_test_helpers:render_and_load_app_config(emqx_conf),
-    ok = emqx_common_test_helpers:start_apps([
-        emqx_conf, emqx_rule_engine, emqx_bridge, emqx_mongodb
-    ]).
+    ok = emqx_common_test_helpers:start_apps(
```

Comment:
Nit: maybe we could update this suite to use `emqx_cth_suite` since we're touching it?

Reply:
I'd rather have that as a separate PR

Reply:
It's a couple lines of change...  The lines that should change are already in this PR, but just adding newlines...